### ISSUE TITLE
export / import data

### ISSUE THREAD

**Issue by NVT1986** [jerameel/sushi#52]
good day!

i thank you kindly for making this very nice and simple app. i used moneywallet for a long time but i realised i needed a much simpler app. then i found Sushi.

however, there is one thing i would really like: export and import of all transactions. for when switching phone or ROM :) i would appreciate it if somebody can implement it.

with kind regards

**Comment by Wurzelkoch** [comment]
There is already an "export" button, but it doesn't seem to work for me. (Version 1.13, running on lineageOS 17.1, which is Android 10 conform)

**Comment by Darkinx** [comment]
Yes, there's an "export" button and it doesn't work for me as well.  (Version 1.14, running on EMUI 12, which is also Android 10 on my Huawei Mate 20). Please Fix and also have an import option if that's possible.

**Comment by dannyhand** [comment]
I would use this app every day if I could export a backup of my transactions.

**Comment by sipek2k** [comment]
Samsung S21 - works (version 1.15) -> /storage/emulated/0/Download/
Thank you !

**Comment by carlobuccisano** [comment]
For me export (into csv) works well. I would also like to see an "Import" function, if possible.
